deps/childprocess.lua

```diff
@@ -164,4 +164,71 @@ local function spawn(command, args, options)
   return em
 end
 
+local function exec(command, args, options, callback)
+  local file
+  local stringArgs = command..table.concat(args, ' ')
```

Comment:
I think you're confusing `exec` with `execFile`.  In node, `exec` takes `(command, options, callback)` and runs the command in a subshell, but `execFile` takes `(file, args, options, callback)` and does not use a subshell.